styles.css

```diff
@@ -1,3 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+.main {
+  max-width: 600px; /* be 100% width, until 600px */
+  margin: 0 auto; /* 0 top and bottom margin, auto left and right margin */
+}
+
+
 body {
-  background-color: white;
+  background-color: #F0F0F0;
+  font-family: sans-serif;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+
+#header {
+  text-align: center;
+  color: maroon; 
+}
+
+#sub-header {
+  text-align: center;
+  color: #051937; 
+}
+
+.container {
+  display: flex; 
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+
+.project-item + .project-item {
```

Comment:
I think this here seems a bit redundant, could just include it together with the other styles for .project-item